MerlinAU.sh

```diff
@@ -5090,9 +5147,8 @@ then FW_RouterModelID="${FW_RouterProductID}"
 else FW_RouterModelID="${FW_RouterProductID}/${GRNct}${MODEL_ID}${NOct}"
 fi
 
-FW_InstalledVers="$(_GetCurrentFWInstalledShortVersion_)"
-FW_NewUpdateVersion="$(_GetLatestFWUpdateVersionFromRouter_ 1)"
-FW_InstalledVersion="${GRNct}$(_GetCurrentFWInstalledLongVersion_)${NOct}"
+FW_InstalledVersion="$(_GetCurrentFWInstalledLongVersion_)"
```

Comment:
Approved the changes to long version, also approving the removal of FW_NewUpdateVersion="$(_GetLatestFWUpdateVersionFromRouter_ 1)"

Seems unrequired and just additional calls for nothing.

Reply:
With all the changes to use the long version, do we really need the short version much anymore? I'm assuming it can be removed correct?

Reply:
> With all the changes to use the long version, do we really need the short version much anymore? I'm assuming it can be removed correct?
>
Done in latest commit https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/226/commits/e49ae4a42dbd7cc0081ebeabd8c915f7aa96d52f

Reply:
Your fantastic Martinski, approved!